src/routing/index.js

```diff
@@ -35,6 +39,14 @@ const Routing = () => {
       <Route exact path={PATHS.AFE} component={AFEpage} />
       <Route exact path={PATHS.PROFILE} component={Profile} />
```

Comment:
No. They have told us to keep AFE open route so that anyone can reach this page. And Profile should be a private route, we may have done it conditionally. It should not be conditionally, making the changes. Thanks to drag the attention here.